README.md

```diff
@@ -0,0 +1,79 @@
+# Conduit Connector Zendesk
+
+### General
+
+The Zendesk connector is one of [Conduit](https://github.com/ConduitIO/conduit) custom plugins. It provides both, a source
+and a destination zendesk connectors.
+
+### How to build it
+
+Run `make`.
+
+### Testing
+
+Run `make test` to run all the tests.
+
+## Zendesk Source
+
+The Zendesk client connector will connect with Zendesk API through the `url` constructed using subdomain specific to individual organization. Upon successful configuration with `zendesk.userName ` and `zendesk.apiToken` the tickets from the given domain is fetched using cursor based [incremental exports](https://developer.zendesk.com/api-reference/ticketing/ticket-management/incremental_exports/) provided by zendesk. The cursor is initiated with start_time set to `0` or the time set in `position` of last successfully ack'd record and all subsequent iterations are done using `next_url` returned by the zendesk till the pipeline is paused. On resuming of the pipeline updated_at time of the last fetched ticket is used to restart the cursor.
+
+### Generating API token in Zendesk
+The api token for the zendesk can be created through zendesk portal by logging in as admin. Refer the zendesk [documentation](https://support.zendesk.com/hc/en-us/articles/4408889192858-Generating-a-new-API-token#topic_bsw_lfg_mmb) for step-by-step setup.
+
+### Change Data Capture (CDC)
+The connector uses the zendesk [cursor based incremental exports](https://developer.zendesk.com/api-reference/ticketing/ticket-management/incremental_exports/) to listen iterate over tickets changed after the given `start_time`. 
+We initiate a `cursor` at the start of the pipeline using the `start_time` as 0, which means we start fetching all the tickets from the start. The subsequent data is fetched using the `after_url` received as part of response.
+When the pipeline resumed after pause/crash, we use the position of the last successfully read record to restart the cursor using the last_modified_time data from position as the start_time.  
```

Comment:
I see that this description is a bit different from the description on line 18 above. E.g. here it's after_url, above it's next_url. Here it's last_modified_time above it's updated_at.

Reply:
done

Reply:
So the difference after_url vs next_url is expected?

Reply:
after_url is the response given from zendesk and it is wrapped in the connector as next_url for reference